source/dotnet/Samples/WPFVisualizer/MainWindow.xaml.cs

```diff
@@ -500,5 +487,10 @@ private void loadTemplateDataButton_Click(object sender, RoutedEventArgs e)
             }
             templateDataTextBox.Text = templateData;
         }
+
+        private void renderCard_Click(object sender, RoutedEventArgs e)
```

Comment:
why is this change being made, I'm understanding you're adding a Button to perform the rendering rather than doing the rendering automatically?

Reply:
when you try to debug, and want to modify json, our current app will constanly render the app. If changes can be made in 1 second, this is somewhat tolerable, but if we have to make changes in multiple places, rendering takes very long time, it's very painful experience. I made changes that we make all the change we want, and just press the button to render.

---
In reply to: [494636503](https://github.com/microsoft/AdaptiveCards/pull/4800#discussion_r494636503) [](ancestors = 494636503)

Reply:
I understand that this can be a bother, we could provide an option to render the card automatically in the future

Reply:
I also believe that if we have this button and don't move the focus into the rendered card, the Accessibility team will file a bug on us, could you make that change?

Reply:
i will back out this particular change, and make the change with the accessiblity change in the future PR. thanks

---
In reply to: [496179405](https://github.com/microsoft/AdaptiveCards/pull/4800#discussion_r496179405) [](ancestors = 496179405)